assets/course-theme/contact-teacher.js

```diff
@@ -31,7 +31,6 @@ export function submitContactTeacher( ev ) {
 		'button.sensei-contact-teacher-form__submit'
 	);
 	submitButton.classList.add( 'sensei-course-theme__button', 'is-busy' );
-	submitButton.disabled = true;
```

Comment:
Is there a good reason to not disable the button here?
If we test clicking 2 times in the button now, it submits the form twice. If we really can't disable the button here, I think we need to create some other approach as a workaround so the user doesn't submit the form twice.

Reply:
The reason I removed it here is the button is focused when it is clicked/triggered. And by being the activeElement it captures the `keydown` event in `sensei-modal.js` that we use to close the modal on `Escape` key. But since the button element is "disabled" the `keydown` event on it is not triggered.

I guess I was a bit quick to simply remove the disable attribute. Probably adding `submitButton.blur()` could've worked fine. You can try that on your end @renatho. Should be a quick fix.

Reply:
I added a focus in the close after the submit. Does it make sense? https://github.com/Automattic/sensei/pull/4665/commits/e877c00e828b1ac5c02672c93b14842577ced39b

Reply:
Yep. I think that's a great solution!